django/db/models/query.py

```diff
@@ -314,12 +315,14 @@ def __or__(self, other):
     # METHODS THAT DO DATABASE QUERIES #
     ####################################
 
-    def iterator(self):
+    def iterator(self, chunk_size=None):
```

Comment:
Is there a reason you used `None` here rather than `constants.GET_ITERATOR_CHUNK_SIZE`? As I was amending the docs with 'The default of ``None`` uses a value of 100." I thought further explanation about that reasoning might be useful.